lib/package.json

```diff
@@ -68,6 +68,7 @@
 		"build:esm": "node --no-warnings --loader ts-node/esm ../scripts/build.ts",
 		"build:cjs": "tsc --module commonjs --outDir dist/cjs",
 		"test": "node --no-warnings --loader ts-node/esm --test ./test/**/*.test.ts",
+		"test:unit": "node --no-warnings --loader ts-node/esm --test ./src/cmaf/test/*.test.ts",
```

Comment:
We are putting cmaf specifically here because is the only one that has unit tests today, but thinking if we should put more generic the path 🤔